Remove double-negative in CORS-preflight fetch credentials tags

jeffcarp opened the pull request:
Resolves #279 

First pull request - happy to change anything. Thanks for adding the `good first bug` label to issues!

annevk commented on Overview.src.html:
```diff
@@ -3406,9 +3406,9 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
-     "<code title>include</code>", and true otherwise
+     <dd>True if <var>request</var>'s
+     <span title=concept-request-credentials-mode>credentials mode</span>
+     is "<code title>include</code>", false otherwise
```
Please keep "and" here and below.

And feel free to add your name to the list of Acknowledgements.

annevk commented:
Oh also, one other minor request I had is if you could stick to the wrapping guidelines. This text is old so might not conform to it yet, but it would be great to make it so as part of this commit. I'm happy to clean that up as part of merging however. Just a thought.

jeffcarp commented:
Added "and" and added myself to the acknowledgements, thanks! 👍

I'm happy to make the changes to conform to the wrapping guidelines but I can't exactly figure out what needs to be changed.

annevk commented:
Thank you, landed as https://github.com/whatwg/fetch/commit/f9a540bd002679bfa238f2fddf5543e7ac9b0d11.

jeffcarp commented:
🎆 🎉 yay thank you!
